t9n/it.coffee

```diff
@@ -103,6 +107,9 @@ it =
       "User not found": "Utente non trovato"
       "Verify email link expired": "Link per la verifica dell'email scaduto"
       "Verify email link is for unknown address": "Il link per la verifica dell'email fa riferimento ad un indirizzo sconosciuto"
+      "At least 1 digit, 1 lowercase and 1 uppercase": "Almeno un numero, 1 carattere minuscolo e 1 maiuscolo"
```

Comment:
This might be too specific, since "1"s could be easily changed from app to app...